index.html

```diff
@@ -59,19 +64,124 @@ <h3>
                 </p>
             </div>
 
+        </section>
+
 
+        <ul class="socials container">
+            <li><a href="#"><img src="images/github.png" alt="github icon"></a></li>
+            <li><a href="#"><img src="images/linkedin2.png" alt="linkedin icon"></a></li>
+            <li><a href="#"><img src="images/two.png" alt=""></a></li>
+            <li><a href="#"><img src="images/twitter.png" alt="twitter icon"></a></li>
+            <li><a href="#"><img src="images/medium.png" alt="medium logo"></a></li>
+        </ul>
+        <img class="back-pic" src="images/background-image1.png" alt="background image">
+    </div>
 
+    <div class="grid-container">
 
-            <ul class="socials container">
-                <li><a href="#"><img src="images/github.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="images/linkedin2.png" alt="linkedin icon"></a></li>
-                <li><a href="#"><img src="images/two.png" alt=""></a></li>
-                <li><a href="#"><img src="images/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="#"><img src="images/medium.png" alt="medium logo"></a></li>
+        <section id="portfolio">
+            <p class="portfolio-heading">My Recent Work</p>
```

Comment:
- [ ] In keeping with semantic principles, this header should be appropriately wrapped using the **`hx`** tag, not **`p`** tag. Kindly refactor.